YoCode/Checks/UnitConverterCheck.cs

```diff
@@ -157,17 +158,24 @@ public Task<List<FeatureEvidence>> Execute()
                     InitializeDataStructures(htmlCode);
                     actual = fetcher.GetActualValues(texts, actions);
 
-                    if (OutputsAreEqual())
+                    if (actions.Count == 0)
                     {
-                        UnitConverterCheckEvidence.SetPassed(new SimpleEvidenceBuilder(unitConverterResultsOutput.ToString()));
+                        UnitConverterCheckEvidence.SetInconclusive(new SimpleEvidenceBuilder(unitConverterResultsOutput.ToString()));
```

Comment:
I think it if is inconclusive it should output some information about why. I'm not sure what the reason would be if the reason for no actions is that it couldn't find what it was looking for in the UI, it should say that. It would help the user know what happened.